bert_e/templates/queue_build_failed.md

```diff
@@ -0,0 +1,33 @@
+{% extends "message.md" %}
+
+{% block title -%}
+Queue build failed
+{% endblock %}
+
+{% block message %}
+
+The corresponding build for the queue failed:
+
+- Checkout the [status page]({{ frontend_url }}).
+- Identify the failing build and review the logs.
+- If no issue is found, re-run the build.
```

Comment:
or no need if there is a more recent green build on the same column.
But I don't know if we want to cover all cases here

Reply:
That's a good point, I would say let's start like this, and if users get confused we'll adapt. We know where the file is it's only a matter of editing this template